### PR TITLE
84 rm add programming meme

### LINKED ISSUE #84: Add Rohan's meme

### PR DESCRIPTION
**Pull request by ROMENDI**
closes #84

### CODE REVIEW

**Review by ahuynh3a — COMMENTED** [review]
There's a conflict, could you resolve it?

File: README.md [hunk 1]
```diff
@@ -55,8 +55,10 @@ Clone this repository (or open in codespace). Create an issue with a meme descri
...
+| ![](assets/writing-code-meme.jpeg) | [Rohan Mendiratta](https://github.com/romendi) | [source](https://www.boredpanda.com/programmer-funny-coding-jokes/) | 
 | ![](assets/darksouls_meme.jpeg) | [David Le](https://github.com/DavidVLe1) | [source](https://www.google.com/url?sa=i&url=https%3A%2F%2Fimgflip.com%2Fmemegenerator%2F171457391%2FYhorm-Dark-Souls&psig=AOvVaw1NXRP49sD037Y1cSpo51iu&ust=1709914567493000&source=images&cd=vfe&opi=89978449&ved=0CBMQjRxqFwoTCLDg7aTG4oQDFQAAAAAdAAAAABAJ) |
 | ![](https://qph.cf2.quoracdn.net/main-qimg-d370a2c0c5d9b4651897ee5e20c524ff) | [John Paul Fababaer](https://github.com/jpfababaer) | [source](https://qph.cf2.quoracdn.net/main-qimg-d370a2c0c5d9b4651897ee5e20c524ff)
 | ![](assets/ap-programmer-life.gif) | [Andrew Pham](https://github.com/andrewp8) | [source](https://media0.giphy.com/media/2KAGlmkPywhZS/200w.gif?cid=6c09b952a3qmebfwsezajf8hjdvs5pgwqp1ahbe4mjodsm1y&ep=v1_gifs_search&rid=200w.gif&ct=g) |
-| ![](assets/listening.jpeg)  | [Jose Pantoja Jr](https://github.com/jpanto21) | [source](https://www.linkedin.com/company/javascript-developer/posts/?feedView%253Dall)|
```
Comment (ahuynh3a) [inline README.md]: I think we can delete this jose

**Review by ahuynh3a — APPROVED** [review]
Looks good!

**Review by reidaumann — APPROVED** [review]
Your meme looks good, but there are merge conflicts to resolve first

**Review by ahuynh3a — APPROVED** [review]

**Pull request merged**